README.md

```diff
@@ -1 +1,366 @@
-# open-days-registration-application-back-end
\ No newline at end of file
+# General description
+
+The repostiory contains an MVC based [Spring Boot](https://spring.io) REST API application using JAVA programming language and scripts for building a portable [Docker](https://www.docker.com/) container, which runs the application.
```

Comment:
I would add the direct link to spring-boot home: https://spring.io/projects/spring-boot

Reply:
Ok updated